scripts/ci/publish.ts

```diff
@@ -321,7 +321,21 @@ async function getCurrentPatchForPatchVersions(patchMajorMinor: { major: number;
   // "3.0.1",
   // "3.0.2"
   // ]
-  let versions = JSON.parse(await runResult('.', 'npm view @prisma/client@* version --json'))
+
+  // We retry a few times if it fails
+  // npm can have some hiccups
+  const remoteVersionsString = await pRetry(
```

Comment:
Do you think adding a few seconds delay could help too?

Reply:
It already has an exponential back off from what I know, it's using the default of https://github.com/tim-kos/node-retry#retrytimeoutsoptions